plugins/oauth/server/constants.py

```diff
@@ -20,9 +20,18 @@
 
 # Constants representing the setting keys for this plugin
 class PluginSettings:
+    PROVIDERS_ENABLED = 'oauth.providers_enabled'
+    GITHUB_CLIENT_ID = 'oauth.github_client_id'
+    GITHUB_CLIENT_SECRET = 'oauth.github_client_secret'
     GOOGLE_CLIENT_ID = 'oauth.google_client_id'
     GOOGLE_CLIENT_SECRET = 'oauth.google_client_secret'
 
+GITHUB_AUTH_URL = 'https://github.com/login/oauth/authorize'
```

Comment:
Do you think we should move these provider-specific constants to the individual provider files? Once we have a half dozen different providers, the current layout could get a little messy, considering they're not properly namespaced.